src/lavinmq/vhost.cr

```diff
@@ -111,6 +111,7 @@ module LavinMQ
       headers = msg.properties.headers
       find_all_queues(ex, msg.routing_key, headers, visited, found_queues)
       headers.delete("BCC") if headers
+      return false unless @replicator.has_min_followers? || @replicator.has_max_lag?
```

Comment:
I think that instead of returning we should block and wait

Reply:
Did a while/sleep thing instead of returning, maybe there is another way to do it? saw that we use select/when in some places to wait, but might only be good for multiple outcome scenarios.. can't find good docs in the crystal api